python/ml4ir/base/model/relevance_model.py

```diff
@@ -516,6 +520,10 @@ def on_predict_end(self, logs):
 
         callbacks_list.append(DebuggingCallback())
 
+        # Weights and Biases Callback
```

Comment:
"Add callback to log Weights and Biases"?
(let's make is a phrase instead of a title :))

Reply:
Done.